modules/apps/29-fee/keeper/msg_server.go

```diff
@@ -37,11 +37,11 @@ func (k Keeper) PayPacketFee(goCtx context.Context, msg *types.MsgPayPacketFee)
 		return nil, channeltypes.ErrSequenceSendNotFound
 	}
 
-	packetId := &channeltypes.PacketId{
-		PortId:    msg.SourcePortId,
-		ChannelId: msg.SourceChannelId,
-		Sequence:  sequence,
-	}
+	packetId := channeltypes.NewPacketId(
```

Comment:
nit: I think we should update all occurrences of local vars `packetId` to `packetID` to align with other names such as `portID`..etc

Reply:
Should we update the channeltype as well then? Either way, follow up PR :+1:

Reply:
please make an issue with your suggestion

Reply:
https://github.com/cosmos/ibc-go/issues/742 👍

Reply:
The actual type cannot be `PacketID` because gRPC gateway doesn't work with `ID`, it has to be `Id`. But I do agree with variables being `packetID`